src/features/upload-documents.feature

```diff
@@ -102,7 +102,7 @@ Feature: upload documents
             # | metric   |
             | land-ownership  |
 
-    Scenario: I can upload all allowed filetypes
+    Scenario Outline: I can upload all allowed filetypes
```

Comment:
this isn't a scenario outline its just a datatable... however, if you made the change to fix the tests running on the pipeline then no worries.